assets/src/editor/events/dates-and-times/editor-date/actions-menu/editor-date-entity-actions-menu.js

```diff
@@ -92,8 +92,8 @@ class EditorDateEntityActionsMenu extends Component {
 	 * @param {boolean} ticketEntitiesLoaded
 	 * @return {IconMenuItem}    View Tickets for Event Date IconMenuItem
 	 */
-	viewTicketsMenuItem = ( dateEntity, ticketEntities, ticketEntitiesLoaded ) => {
-		const tooltip = ticketEntitiesLoaded && isEmpty( ticketEntities ) ?
+	viewTicketsMenuItem = ( dateEntity, ticketEntities ) => {
```

Comment:
`ticketEntitiesLoaded` is no longer needed because this component won't be mounted until the editor is initialized.  So we can just use the `ticketEntities` check here.